### PR TITLE
PR

### PR DESCRIPTION
**Pull request by oscarmild**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,4 @@
 # active-task-manager
 active-task-manager
 
-
-
-
-
-
+<p><blockquote>Express is a light-weight web application framework to help organize your web application into an MVC architecture on the server side. TypeScript enables us to develop a solid web applications using Express on Node.js.</blockquote></p>
```
Comment (fromgopi) [inline README.md:4]: @oscarmild 

Why dont you complete the whole readme.?

File: webpack.config.js [hunk 2]
```diff
@@ -2,18 +2,20 @@ const nodeExternals = require('webpack-node-externals');
 const path = require("path");
 const webpack = require('webpack');
 const { CleanWebpackPlugin } = require("clean-webpack-plugin")
-const CopyWebpackPlugin = require("copy-webpack-plugin");
+const CopyWebpackPlugin = require("copy-webpack-plugin"); 
 
 module.exports = (env, argv) => {
     return {
+
```
Comment (fromgopi) [inline webpack.config.js:9]: @oscarmild Is there any update on the webpack file?

File: webpack.config.js [hunk 3]
```diff
@@ -2,18 +2,20 @@ const nodeExternals = require('webpack-node-externals');
...
         entry: {app: './index.ts'},
+
         externalsPresets: { node: true },
         context: __dirname,
 
         externals: [nodeExternals()],
         output: {
             path: path.join(__dirname, "dist"),
-            filename: "atm-v-01.js"
+            filename: "active-task-v-01.js"
```
Comment (fromgopi) [inline webpack.config.js:18]: @oscarmild Please make sure that you read all the constant strings from the constant files or .env

Comment (fromgopi) [inline webpack.config.js:18]: @oscarmild Please add some more plugins for reducing output file

**Review by fromgopi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (oscarmild) [reply README.md:4]: @fromgopi We will write by next pr

File: webpack.config.js (hunk 2 above)
Comment (oscarmild) [reply webpack.config.js:9]: @fromgopi Yes we will update the whole file which can build and deploy into the server

File: webpack.config.js (hunk 3 above)
Comment (oscarmild) [reply webpack.config.js:18]: @fromgopi  sure We will move into constant files

Comment (oscarmild) [reply webpack.config.js:18]: @fromgopi it will be in other branch

**Review by fromgopi — APPROVED** [review]
@oscarmild Approved

**Pull request merged**